aibolit/patterns/method_chaining/method_chaining.py

```diff
@@ -0,0 +1,41 @@
+import javalang
+
+from aibolit.utils.utils import remove_comments
+
+
+class MethodChainFinder:
```

Comment:
@lyriccoder OO classes souldn't have names ending in -er. Please take a look at [this article](https://www.yegor256.com/2015/03/09/objects-end-with-er.html)